### PR TITLE
config: add internal() option to src, dst, rewrite, parser

### PR DESCRIPTION
**Pull request by alltilla**
It puts the registered stats counters to level 3.

See #4453 for example metrics.

Depends on #4447

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE

**Review by MrAnno — DISMISSED** [review]

**Comment by MrAnno** [comment]
@kira-syslogng do stresstest

File: modules/affile/affile-dest.c [hunk 1]
```diff
@@ -294,8 +294,13 @@ affile_dw_set_owner(AFFileDestWriter *self, AFFileDestDriver *owner)
 
   if (self->owner)
     log_pipe_unref(&self->owner->super.super.super);
+
   log_pipe_ref(&owner->super.super.super);
   self->owner = owner;
+
+  if (!owner)
+    return;
+
```
[the remark this answers is not in the document]
Comment (alltilla) [reply modules/affile/affile-dest.c]: Dropped this, we will fix this properly in a separate PR.

**Comment by alltilla** [comment]
@kira-syslogng do stresstest

**Comment by kira-syslogng** [comment]
Kira-stress-test: Build SUCCESS

**Review by MrAnno — APPROVED** [review]

**Pull request merged**